demo/api/schema.gql

```diff
@@ -68,6 +68,21 @@ type DamFile {
   fileUrl: String!
   duplicates: [DamFile!]!
   damPath: String!
+  dependents: [BlockIndexDependency!]!
+}
+
+type BlockIndexDependency {
```

Comment:
I don't like this type. Maybe we can come up with a different approach

Reply:
What do you have in mind? 

I first thought about a union type that directly returns the dependency/dependent objects. But I see no way how we could do that considering the dependents could be defined in the application

Reply:
and I would try to make only those parts public thru the api that we actually need